internal/test/block.go

```diff
@@ -91,33 +90,3 @@ func MakeHeader(t *testing.T, h *types.Header) *types.Header {
 
 	return h
 }
-
-func MakeBlock(state sm.State) *types.Block {
```

Comment:
MakeBlocks, MakeVote, MakeVoteSet tend to have multiple occurrences across our code, I've done some quick tidy-up in the last months, but any improvement (such as the one done here) is super-welcome :-)